### PR TITLE
[i18n] Prepare stat var titles for translation

### PR DESCRIPTION
**Pull request by beets**
- Add a new json for stat var title messages, including descriptions. As discussed in group thread, formatting for some titles were updated, as well as many bugs fixed
- Also fixed issues with other newly extracted messages while preparing import to TC

### CODE REVIEW

File: server/l10n/messages.pot [hunk 1]
```diff
@@ -207,6 +190,41 @@ msgstr ""
 msgid "Gross National Income"
 msgstr ""
 
+#. Blog post entry. Please take care to maintain valid <a> tags in translation. See this string in action at https://datacommons.org
+#: server/templates/static/homepage.html:56
+msgid "HOME-BLOG_POST_1"
+msgstr "%(project)s is now accessible on Google Search! <a href=%(blog_url)s>Read more here</a>"
+
+#. Describes building your own tools and analysis using our API (here Python refers to the programming language - see python.org, and REST is an acronym for REST is an acronym for REpresentational State Transfer). Please see this string in action at https://datacommons.org
```
Comment (shifucun) [inline server/l10n/messages.pot:198]: remove

File: server/l10n/messages.pot [hunk 2]
```diff
@@ -207,6 +190,41 @@ msgstr ""
...
+#: server/templates/static/homepage.html:143
+msgid "HOME-BUILD_CALLOUT"
+msgstr "Use the Python and REST API's to do your own custom analysis, build your own tools and more."
+
+#. A header of a section showing the many data sources included in the Data Commons Graph. See this string at https://datacommons.org
+#: server/templates/static/homepage.html:176
```
Comment (shifucun) [inline server/l10n/messages.pot:204]: curious if the line content changes, does it matter?

File: server/l10n/messages.pot [hunk 3]
```diff
@@ -261,25 +275,25 @@ msgstr ""
...
+msgstr ""
 
 #. The name of a tool to browse statistics about a place.
 #: server/templates/base.html:72 server/templates/base.html:122 server/templates/place.html:19 server/templates/place.html:20 server/templates/place_landing.html:19 server/templates/place_landing.html:20 server/templates/place_landing.html:38
 msgid "Place Explorer"
-msgstr ""
+msgstr "<p>The Place Explorer tool helps you understand the core data underlying a place. It pulls together a set of charts on the demographics, education, economics, etc., of the place, placing them in context with comparisons to other places. You can also see how different places rank along different metrics line unemployment, crime and income.</p><p>You can dive into the charts for your favorite city, zipcode, county, state or country using the search box above.</p><p>Or, start exploring some well known places ...</p>"
```
Comment (shifucun) [inline server/l10n/messages.pot:296]: line -> like

File: server/l10n/messages.pot (hunk 2 above)
Comment (beets) [reply server/l10n/messages.pot:204]: It doesn't matter - this file is automatically generated. I've found it helpful to see the pointer to the source of the string though.

File: server/l10n/messages.pot (hunk 1 above)
Comment (beets) [reply server/l10n/messages.pot:198]: Sorry, what needs to be removed? This string is needed for the homepage. The translators also want very detailed descriptions, so it's easier to just point them to the homepage so they can see the context.

File: server/l10n/messages.pot (hunk 3 above)
Comment (beets) [reply server/l10n/messages.pot:296]: wow, nice catch!

**Review by beets — COMMENTED** [review]
Thanks for the review!

File: server/l10n/messages.pot (hunk 1 above)
Comment (shifucun) [reply server/l10n/messages.pot:198]: this sentence: "and REST is an acronym for REST is an acronym for REpresentational State Transfer)"

**Review by shifucun — APPROVED** [review]

File: server/l10n/messages.pot (hunk 1 above)
Comment (beets) [reply server/l10n/messages.pot:198]: wow, thanks for that catch! i really couldn't see that.

**Pull request merged**